Tests/varLib/varLib_test.py

```diff
@@ -229,37 +229,51 @@ def test_varlib_gvar_explicit_delta(self):
 
     def test_varlib_build_CFF2(self):
         ds_path = self.get_test_input('TestCFF2.designspace')
-        suffix = '.otf'
-        expected_ttx_name = 'BuildTestCFF2'
-        tables = ["fvar", "CFF2"]
+        ttx_dir = self.get_test_input("master_cff2")
+        expected_ttx_path = self.get_test_output("BuildTestCFF2.ttx")
 
-        finder = lambda s: s.replace('.ufo', suffix)
-        varfont, model, _ = build(ds_path, finder)
-        # some data (e.g. counts printed in TTX inline comments) is only
-        # calculated at compile time, so before we can compare the TTX
-        # dumps we need to save to a temporary stream, and realod the font
+        self.temp_dir()
+        for path in self.get_file_list(ttx_dir, '.ttx', 'TestCFF2_'):
+            print(path)
```

Comment:
not sure if this print(path) was intentionally left here. pytest would capture stdout by default, only shows it when a test fails.